sources/packages/web/src/components/aest/students/ApplicationHeaderTitle.vue

```diff
@@ -0,0 +1,63 @@
+<template>
```

Comment:
I would prefer this component to be built under common and not `components/aest/*` as we could potentially have this header for students(most certainly) and institutions(may be).

Reply:
The team / ticket decision was "Create the component inside the components/aest/students folder." I think we should stick with that unless / until the header is requested for different users.

Reply:
True but as we figured something in recent that this may potentially used for students, I asked here. 

We can quickly check with @andrewsignori-aot

Reply:
@dheepak-aot the mentioned one for students already exists on other students' pages.
We will need to decide later if the same component will be used or if this one will be shared.
If the student one is already a different component I would not change the decision on the technical AC for this one.